01-node-tutorial/answers/01-intro.js

```diff
@@ -0,0 +1,2 @@
+console.log("Hello world");
```

Comment:
Not sure how if this needed to be included in a previous branch or if you need to merge your old changes to main but this should have been for previous weeks. Did you want me to review them? Not sure.